app/views/application/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <% content_for :title do %>
   Check your CSV files with CSVLint
 <% end %>
+<%= javascript_include_tag "resumable.js" %>
+<%= stylesheet_link_tag "resumable.css" %>
```

Comment:
We could include these into the asset pipeline with in application.js and application.css